src/main/java/com/box/sdk/BoxAIExtractField.java

```diff
@@ -0,0 +1,74 @@
+package com.box.sdk;
+
+import com.box.sdk.internal.utils.JsonUtils;
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+import java.util.List;
+
+public class BoxAIExtractField extends BoxJSONObject {
+    /**
+     * The type of the field. It include but is not limited to string, float, date, enum, and multiSelect.
+     */
+    private String type;
+    /**
+     * The description of the field.
+     */
+    private String description;
+    /**
+     * The display name of the field.
+     */
+    private String displayName;
+    /**
+     * The key of the field.
+     */
+    private String key;
+    /**
+     * A list of options for this field.
+     * This is most often used in combination with the enum and multiSelect field types.
+     */
+    private List<BoxAIExtractFieldOption> options;
+    /**
+     * The prompt of the field.
+     */
+    private String prompt;
+
+    /**
+     * Constructs a BoxAIExtractField object with a given type, description, display name, key, options, and prompt.
+     *
+     * @param type        the type of the field.
+     * @param description the description of the field.
+     * @param displayName the display name of the field.
+     * @param key         the key of the field.
+     * @param options     a list of options for this field.
+     * @param prompt      the prompt of the field.
+     */
+    public BoxAIExtractField(String type,
```

Comment:
Not all the fields are required here. It would mean that the customer needs to pass null to avoid them - this is not desired behaviour. Create another constructor only with required fields

Reply:
And verify in runtime if setting only these field actually works (I don't trust schema :) )